app/Http/Controllers/BrevoController.php

```diff
@@ -61,12 +65,150 @@ public function __construct()
+     *             'X-Google-DKIM-Signature' => 'v=1; a=rsa-sha256; c=relaxed/relaxed; d=1e100.net; s=20230601; t=1711214316; x=1711819116; h=to:subject:message-id:date:from:mime-version:x-gm-message-state :from:to:cc:subject:date:message-id:reply-to; bh=eBSl5M0zvmTd+dFXGXMMSWrQ4nCvUdyVx+1Xpl+YuX8=; b=fg4tXZnstRBexYlC6MD7C7is0kQj+xY66cSJ78tSa7PtSFQzY0zajDMsepMCGiiWmN /Pc/tRtk53pru/OtfzRT9pbM6mhM1arIt+QaQBQGU5xZVV5JXfPmdnPzXqAbQztyeHrk UcEkz+qDN3JNoidw2dJhhdt5MxdKssR572NwtBrn/rN7f1o/ThWzEz+P0o06GVBpxVYP wM0EkvcJj2SUOcn36kmp1ccbMUwYCU2h1JmniEFY8RTqu2il13iXoBvG4YPxe0c0hJ6z zw1N5rONeQM113N1rpbQzS1QLSngczuOhN24M3TOwrHJIec/BxrOW6KWl/uPUqiZAf65 f0tg==',
+     *             'X-Gm-Message-State' => 'AOJu0YzKhR1HY1oUXoq++LLpl6UOz1S60NfPxuPXBLcP+6aACYle8rqQ fYHe2rQYTpg4KWiOswu858STOW8qmiewXD6gH/LbmEFs7sknRyDPNr/+L0cv828A3o+SOvXu3uP SY6H1aNSwIpqTRhJ+nNjTuSUpuSoABd9fYXFwPuivV0DtBhoVmpE=',
+     *             'X-Google-Smtp-Source' => 'AGHT+IHdA9ZhW0dQxgOYx2OXBGmu4pzSR/zwJ0vcPNXFSqttKCPS2oTw1a9b2mMdhyUeoRAwP5TmhHlAtqUUrOPwkgg=',
+     *             'X-Received' => 'by 2002:a50:d74c:0:b0:567:3c07:8bbc with SMTP id i12-20020a50d74c000000b005673c078bbcmr2126401edj.21.1711214316135; Sat, 23 Mar 2024 10:18:36 -0700 (PDT)',
+     *             'MIME-Version' => '1.0',
+     *             'From' => 'Max Mustermann <max@mustermann.de>',
+     *             'Date' => 'Sat, 23 Mar 2024 18:18:20 +0100',
+     *             'Message-ID' => '<CADfEuNvumhUdqAUa0j6MxzVp0ooMYqdb_KZ7nZqHNAfdDqwWEQ@mail.gmail.com>',
+     *             'Subject' => 'TEST',
+     *             'To' => 'test@test.de',
+     *             'Content-Type' => 'multipart/mixed',
+     *         ),
+     *         'SpamScore' => 2.8,
+     *         'ExtractedMarkdownMessage' => 'TEST',
+     *         'ExtractedMarkdownSignature' => NULL,
+     *         'RawHtmlBody' => '<div dir="ltr">TEST</div>',
+     *         'RawTextBody' => 'TEST',
+     *         'EMLDownloadToken' => 'eyJmb2xkZXIiOiIyMDI0MDMyMzE3MTgzNi45OS43OTgwMDM4MDQiLCJmaWxlbmFtZSI6InNtdHAuZW1sIn0',
+     *         ),
+     *     ),
+     *   )
+     */
+    public function inboundWebhook(Request $request)
+    {
+        $input = $request->all();
+
+        if (!($request->has('token') && $request->get('token') == config('ninja.inbound_mailbox.inbound_webhook_token')))
+            return response()->json(['message' => 'Unauthorized'], 403);
+
+        if (!array_key_exists('items', $input)) {
+            Log::info('Failed: Message could not be parsed, because required parameters are missing.');
+            return response()->json(['message' => 'Failed. Invalid Parameters.'], 400);
+        }
+
+        foreach ($input["items"] as $item) {
+
+            if (!array_key_exists('Recipients', $item) || !array_key_exists('MessageId', $item)) {
+                Log::info('Failed: Message could not be parsed, because required parameters are missing. At least one item was invalid.');
```

Comment:
can we change all instances of Log::info() to =>  nlog() just for consistency please.